### PR TITLE
Attempts to correct auto-version-bump for torchvision by removing spurious tags

### PR DESCRIPTION
**Pull request by anjos**
PR recommended by @h-vetinari on conda-forge/torchvision-feedstock#61 to re-enabled auto-bumping.

### CODE REVIEW

File: node_attrs/torchvision.json [hunk 1]
```diff
@@ -1802,25 +1802,13 @@
...
...nda/envs/run_env/lib/python3.9/site-packages/ruamel/yaml/composer.py\", line 185, in compose_sequence_node\n    node.value.append(self.compose_node(node, index))\n  File \"/usr/share/miniconda/envs/run_env/lib/python3.9/site-packages/ruamel/yaml/composer.py\", line 143, in compose_node\n    node = self.compose_mapping_node(anchor)\n  File \"/usr/share/miniconda/envs/run_env/lib/python3.9/site-packages/ruamel/yaml/composer.py\", line 223, in compose_mapping_node\n    item_value = self.compose_node(node, item_key)\n  File \"/usr/share/miniconda/envs/run_env/lib/python3.9/site-packages/ruamel/yaml/composer.py\", line 143, in compose_node\n    node = self.compose_mapping_node(anchor)\n  File \"/usr/share/miniconda/envs/run_env/lib/python3.9/site-packages/ruamel/yaml/composer.py\", line 216, in compose_mapping_node\n    while not self.parser.check_event(MappingEndEvent):\n  File \"/usr/share/miniconda/envs/run_env/lib/python3.9/site-packages/ruamel/yaml/parser.py\", line 146, in check_event\n    self.current_event = self.state()\n  File \"/usr/share/miniconda/envs/run_env/lib/python3.9/site-packages/ruamel/yaml/parser.py\", line 611, in parse_block_mapping_key\n    raise ParserError(\n"
```
Comment (h-vetinari) [inline node_attrs/torchvision.json:1811]: If you leave a comma at the end of the line here, the diff will become smaller, and the effect will be the same.

**Review by h-vetinari — COMMENTED** [review]
LGTM, thanks!

File: node_attrs/torchvision.json (hunk 1 above)
Comment (anjos) [reply node_attrs/torchvision.json:1811]: Right, but then it would not be a strictly valid JSON file.  Which parser are we using?

Comment (h-vetinari) [reply node_attrs/torchvision.json:1811]: Ah right, sorry. We're in JSON land and not python. Carry on. 😅

**Comment by anjos** [comment]
@h-vetinari, can you please merge this?  I have no powers here.

**Comment by h-vetinari** [comment]
> @h-vetinari, can you please merge this? I have no powers here.

As it happens, neither do I 🙃

CC @beckermr @jakirkham @ocefpaf

**Comment by beckermr** [comment]
We can merge this but if the bot found the tags before, it likely will again.

**Comment by h-vetinari** [comment]
> We can merge this but if the bot found the tags before, it likely will again.

Thanks. Sometimes spurious tags get pushed and then deleted, let's hope the bot doesn't find them anymore :)

**Comment by h-vetinari** [comment]
Btw, the bot opened https://github.com/conda-forge/torchvision-feedstock/pull/66, so at least for now, this was a success :)

**Pull request merged**